internal/secrets/localsecrets/example_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package localsecrets_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/go-cloud/internal/secrets/localsecrets"
+)
+
+func ExampleEncrypterDecrypterEncrypt() {
+	sk := "I'm a secret string!"
```

Comment:
So this is the key? Maybe call the variable `secretKey`.

Reply:
I have been under the impression that variable names are basically "the shorter the better". If this is incorrect I can certainly make them more expressive, but that contradicts what I have been told to date.

Reply:
If this were an ordinary function in regular code, `sk` would be right. Since it's an example, a longer name is justifiable.

Or you could write
```
sk := "secret key"
```
Basically I was a bit confused because I didn't see "key".